GNUmakefile

```diff
@@ -38,4 +38,5 @@ debug_test:
 	TF_LOG=debug make test
 
 generate-docs:
+	go get github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
```

Comment:
should be 
```yaml
go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs@0.13.0
```


from https://tip.golang.org/doc/go1.16#modules :
> This is useful for installing executables without affecting the dependencies of the main module.

Reply:
change